src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,35 @@
+import { useNavigate } from "react-router-dom";
+import { useState, useEffect } from "react";
+import styles from "./Pagination.module.scss";
+import { PaginationItem } from "./PaginationItem/PaginationItem";
+
+type Props = {
+    routes: Array<string>;
+};
+
+export const Pagination = ({ routes }: Props) => {
+    const [activeRoute, setActiveRoute] = useState(routes[0]);
+
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        navigate(activeRoute);
+    }, [activeRoute]);
+
+    return (
+        <div className={styles.paginationWrapper}>
+            {routes.map((route, index) => {
+                return (
```

Comment:
It is better to extract this into a new function

Reply:
I think it's more practical to put it directly inside if I don't do any logic in between